SwiftmailerBundle.php

```diff
@@ -26,6 +28,11 @@ public function build(ContainerBuilder $container)
         parent::build($container);
 
         $container->addCompilerPass(new RegisterPluginsPass());
+
+        // Older supported versions of Symfony don't have this base class, but they don't have the hot_path optimization either.
+        if (\class_exists('Symfony\Component\DependencyInjection\Compiler\AbstractRecursivePass')) {
```

Comment:
maybe check the ResolveHotPathPass instead ?

Reply:
no. what I need to check is the existing of the parent class of my EnsureNoHotPathPass (to be sure that my own compiler pass won't trigger a fatal error).

The goal is not to detect support for ResolveHotPathPass but for AbstractRecursivePass that I'm using (this comment is explaining that not being able to clear the tag on older version is not an issue, as they won't add it either).

And I had to use the AbstractRecursivePass (or to duplicate it), to ensure that things works even for inlined Swift services.